jsHelper/spicetifyWrapper.js

```diff
@@ -1127,26 +1127,31 @@ Spicetify.SVGIcons = {
 })();
 
 class _HTMLContextMenuItem extends HTMLLIElement {
-	constructor({ name, disabled = false, icon = undefined, divider = false }) {
+	constructor({ name, disabled = false, icon = undefined, actionIcon = undefined, divider = false }) {
 		super();
 		this.name = name;
-		this.icon = icon || "";
+		(this.icon = icon), (this.actionIcon = actionIcon);
```

Comment:
```suggestion
		this.icon = icon;
		this.actionIcon = actionIcon;
```

Reply:
formatting causes this lint will fail otherwise.

Reply:
![image](https://github.com/spicetify/spicetify-cli/assets/9348108/72f43b32-3b02-4d10-b0e1-02aeccb96571)
why would it fail after using prettier on it?

Reply:
I used prettier and it kept making it into that one liner 🤔 used the repo config too

Reply:
![image](https://github.com/spicetify/spicetify-cli/assets/9348108/555626ac-73ed-4c31-b434-df5150a4cef7)

Reply:
i don't disbelieve you, just couldn't get it to format like that on my end. must have been some weird bug. whilst on the topic what are your thoughts on using naming conventions: leadingIcon, trailingIcon?

Reply:
Use the names that Spotify uses